build.assets/tooling/cmd/changelog/README.md

```diff
@@ -0,0 +1,81 @@
+# changelog
+
+This script generates a changelog for a release.
+
+## Usage
+
+```
+usage: changelog [<flags>]
+
+Flags:
+  --[no-]help              Show context-sensitive help (also try --help-long and --help-man).
+  --baseBranch=BASEBRANCH  The base release branch to generate the changelog for. It will be of the form branch/v* ($BASE_BRANCH)
```

Comment:
```suggestion
  --base-branch=BASEBRANCH  The base release branch to generate the changelog for. It will be of the form branch/v* ($BASE_BRANCH)
```